test/gtest.cmake

```diff
@@ -41,10 +41,3 @@ endif()
 
 # Add googletest, defines gtest and gtest_main targets
 add_subdirectory(${CMAKE_CURRENT_BINARY_DIR}/googletest-src ${CMAKE_CURRENT_BINARY_DIR}/googletest-build EXCLUDE_FROM_ALL)
-
-# Remove visibility.h from the compile flags for gtest because of poisoned exit()
```

Comment:
This was only ncessary in the PX4/Firmware since the cmake setup there globally includes a custom visibility.h which poisons some symbols like `exit()` which are necessary to properly compile gtest. I'd presume it's not necessary for the ecl because I hope the cmake structure is smarter here.

Reply:
Cool, I think we should then be good, removing this in ECL.